packages/iterator-utilities/src/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+	"extends": "../../../tsconfig.base.json",
+	"compilerOptions": {
+		"lib": ["dom", "esnext"],
```

Comment:
Does this really need DOM? If not drop the entire field because the base is already esnext.